tests/notus/example.csv

```diff
@@ -0,0 +1,4 @@
+# General metadata:
```

Comment:
Note that this and the corresponding test will have to be updated at least once more after certain small changes to the CSV format.